### PR TITLE
Rebase with keyword

### LINKED ISSUE #5: support keyword to trigger rebase
do rebase only on mention with keyword `@botname keyword`.
it would allow existing bot credentials

### LINKED ISSUE #9: Rebases when comment deleted
The bot rebases twice, once when the comment is created and then when comment of rebase is deleted.

### PR DESCRIPTION
**Pull request by jaggs6**
closes #5 and closes  #9 
- Rebase only with keyword "rebase"
- Rebase only when comment is created

### CODE REVIEW

**Comment by jaggs6** [comment]
@chrisledet @Guria please review

File: http/rebase.go [hunk 1]
```diff
@@ -46,7 +47,7 @@ func Rebase(w http.ResponseWriter, r *http.Request) {
 	}
 
 	go func() {
-		if !github.WasMentioned(event.Comment) {
+		if !(strings.Compare(event.Action, "created") == 0 && github.WasMentioned(event.Comment) && strings.Contains(event.Comment.Body,"rebase")) {
```
Comment (chrisledet) [inline http/rebase.go]: Please add a space between the two arguments of `strings.Contains`

**Comment by chrisledet** [comment]
Thanks for the contributions. Just had a minor comment on one of the changes. I'll merge this once it's fixed.

**Comment by jaggs6** [comment]
@chrisledet done

**Pull request merged**